heclib/heclib_c/heclib_c.vcxproj

```diff
@@ -1,654 +1,656 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I realize it's not a big deal on this file, but try to avoid situations where the diff shows every line changed as it makes it impossible to see the *actual* differences.

This is normally caused by the line endings being changed from CRLF to LF or vice versa during the edit. VS Code or other editors might not show that every line has changed, but the Git GUI utility will do so. If the editor doesn't support changing the line endings (VS Code does), then the dos2unix or unix2dos utilities can be used.

I nearly always review changed files in Git GUI before pushing for this purpose. It's available with Git on Windows, Linux, & Solaris.

Reply:
I agree!  I also 'nearly always review changed files' before pushing.   For example I haven't pushed the VS 2022 project files even though every commit I have to be careful not to.

Reply:
I guess that proves the "nearly always" in my comment doesn't mean "always." The preacher needs to listen to his sermon! :)